gcp-cloud-run/src/endpoint.tf

```diff
@@ -0,0 +1,13 @@
+module "serverless_endpoint" {
```

Comment:
We should consider rolling this out as well. Most of the resources that might need customization are buried in here it looks like. Not needed right now, but something to add to the backlog.